README.md

```diff
@@ -73,17 +73,33 @@ Some validation options can be extended by creating `.doc_gen/validation.yaml`.
 - `sample_files`: Sample files are only allowed with certain names. To allow additional sample files, add their file name (with extension, but not path) to this list.
 
 ## New Releases
+There are 2 stages: testing and deployment.
 
-1. When making a release, find the most recent commit sha (or sha that will be tagged for the release).
-2. Update https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/.github/workflows/validate-doc-metadata.yml to that tag.
-  * Also add that commit sha to the [validation config](https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/.doc_gen/validation.yaml)
-3. Create a PR in aws-doc-sdk-examples (do not submit it) to run tests.
-4. When tests in the main repo are passing, create a tag in this -tools repo at that sha with the format `YYYY-MM-DD-A`, where `YYYY-MM-DD` are the year, month, and day of the release and -A is -A, -B, -C etc for multiple releases within a day.
-5. Push that tag.
-6. Update the PR from 2 with that tag, remove the validation sha exception, and push that for draft.
-7. Create a release using the automated "Create release from tag" button.
-8. Perform internal update process.
+### 1. Testing
 
+1. **Create a testing branch** from [aws-doc-sdk-examples@main](https://github.com/awsdocs/aws-doc-sdk-examples/tree/main).
+2. **Find the most recent commit SHA in [aws-doc-sdk-examples-tools:main](https://github.com/awsdocs/aws-doc-sdk-examples-tools/commits/main/)**.
+3. **Update your testing branch**: Add your SHA as a tag (`org/repo@tag`, e.g. `awsdocs/aws-doc-sdk-examples-tools@e7c283e916e8efc9113277e2f38c8fa855a79d0a`) to the following files:
```

Comment:
"Add your sha as a tag" this doesn't make sense? There are no tags in this step? Tags are created in step 1 part 2